packages/core/src/demo/status-bot.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+//@ts-check
+
+const APIClient = require('@wireapp/api-client');
+const {Account} = require('@wireapp/core');
+const {Config} = require('@wireapp/api-client/dist/commonjs/Config');
+const {MemoryEngine} = require('@wireapp/store-engine');
+
+const {version} = require('../../package.json');
+
+const conversationId = process.argv[2];
```

Comment:
```
CONVERSATION_ARGUMENT_INDEX = 2;
```

Reply:
Please discuss this with @bennyn.

Reply:
The only discussion is what would be the best name for this number 2 :)